sample-customisation/README.md

```diff
@@ -1,4 +1,23 @@
 # Sample Customisation
 
-This sample illustrates how you can override the String and Colors of the [Beacon UI](https://bintray.com/helpscout/beacon/beacon-ui/_latestVersion). It also illustrates using a Secure mode Beacon.    
+This Java based sample illustrates some of the key features you can use to customise the Beacon SDK to add additional data about the
+user and how to override certain Beacon settings.
 
+For more information see the [Android technical documentation](https://developer.helpscout.com/beacon-2/android)
+
+## Add additional meta data about your user using the Beacon
+
+* [identify()](https://developer.helpscout.com/beacon-2/android/#normal-mode) - to provide email and name of the user
+* Pre-populate the “Create a message” contact form data with Prefill
+* Add session Attributes which are added to the Beacon visitor activity Note in the specific conversation the user creates
```

Comment:
💡 Maybe explain that session attributes are added after a message is sent or a chat is created?